src/main/java/Decide.java

```diff
@@ -43,4 +43,44 @@ public static boolean LIC0(Point[] pts, Parameters params) {
         }
         return false;
     }
+
+    /**
+     * Checks distance between two points.
+     * @param pts array of Point objects
+     * @param params Parameter object that includes 
+     * q_pts, number of consecutive points, and QUADS, number of quadrants
+     * @return true if at least one set of Q_PTS consecutive data points lie in more than QUADS quadrants.
+     */
+    public static boolean LIC4(Point[] pts, Parameters params) {
+        int q_pts = params.q_pts;
+        int quads = params.quads;
+        if (q_pts < 2 || q_pts > pts.length)
+            throw new IllegalArgumentException("q_pts should be between 2 and pts.length");
+        if (quads < 1 || quads > 3)
+            throw new IllegalArgumentException("quads should be between 1 and 3");
```

Comment:
I realised that according to the [style guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used) should braces be used where they are optional

Reply:
Fixed